packages/internals/src/utils/handlePanic.ts

```diff
@@ -68,6 +68,20 @@ ${chalk.dim(`Learn more: ${link('https://pris.ly/d/telemetry')}`)}
     }
   }
 
+  /*
+  Example flow:
+
+  1) Prompt: Would you like to submit an error report? (no)
+     Prompt: Would you like to create a Github issue? (yes | no)
+
+  2) Prompt: Would you like to submit an error report? (yes)
+     Would you like to create a Github issue? (automatically yes)
+  */
+
+  // Automatically create a Github issue if we already submitted an error report.
+  //
+  // TODO: does this still make sense? (If we already have an error report, it's better to manually create an issue
+  // from the error reporting dashboard, as it includes the schema file and other details, and it's monitored weekly)
   await wouldYouLikeToCreateANewIssue({
     prompt: !shouldSubmitReport,
```

Comment:
This automatic prompt was introduced by [this commit](https://github.com/prisma/prisma/commit/1841e611d9ab3864ab183f2a7a913229ba3601b3#diff-b18af32f165c908984196d3c78f7747e838e0738216c13ad285a46e109eba161).